dev/sbt-plugin/src/sbt-test/sbt-plugin/projections-happy-path/test

```diff
@@ -18,6 +18,14 @@
 
 > runAll
 
+# Locator and gateway should be up
+> validateRequest retry-until-success http://localhost:9008
+> validateRequest retry-until-success http://localhost:9000
+
+# Ensure the foo service is reachable
+> validateRequest retry-until-success http://localhost:9008/services/hello-scala status 200
+> validateRequest retry-until-success http://localhost:9000/api-java/hello/Alice status 200
```

Comment:
These four statements prevent the test from moving forward until: service locator, service gateway and the actual service is (1) registered and (2) connected to the database.

We don't need all four steps but it's a pattern used in other multiple `scripted` tests I forgot to use here so I copied without simplifying the unnecessary steps.